.github/workflows/image.yml

```
@@ -37,6 +42,9 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/arm/v7
           push: true
           tags: superorbital/wordchain:latest
+      - uses: sigstore/cosign-installer@main
+      - name: Cosign signature
+        run: COSIGN_EXPERIMENTAL=1 cosign sign -a git_sha=$GITHUB_SHA docker.io/superorbital/wordchain:latest
```

```suggestion
        run: COSIGN_EXPERIMENTAL=1 cosign sign --recursive -a git_sha=$GITHUB_SHA docker.io/superorbital/wordchain:latest
```

this will do multi-platform